### DISCUSSION TITLE
Dark and light mode transitions on certain elements look strange

### DISCUSSION THREAD

**Discussion by FireIsGood** [picocss/pico#491] [Help]
When transitioning between light and dark mode, elements such as `input` and `a` have a transition set to the global `--pico-transition` variable. These end up having a really distracting look when swapping between color schemes.

Additionally, the [home page](https://picocss.com/) example doesn't have this issue as it is overwritten by a `transition: border-radius var(--pico-transition)` which feels strange since it should act how a default setup for Pico would work.

I think that the transition for background color should be left off inputs since it doesn't match how the rest of the page shifts in color. If the maintainers agree on these points, I can find the instances of this in the codebase and make a pull request later.

## Current

The input elements (and also `a` elements) have a transition on the background color except on the home page.

On the home page:

https://github.com/picocss/pico/assets/109556932/ab4b7ba8-f813-4ba1-aa18-ded0d1e02f04

On any other website:

https://github.com/picocss/pico/assets/109556932/be4bd442-7442-4db6-98dd-87989056cf36

## Proposed

The input elements (and also `a` elements) would have no transition on the background color

https://github.com/picocss/pico/assets/109556932/51142e81-5165-40d7-b111-797916c0a182

## Additional Notes

There is also a FOUC when a color mode different to the user's `prefers-color-scheme` is selected where the screen flashes the wrong color scheme. This appears on the home page, docs, and all the examples which makes the slow background transition for inputs a lot more visible. This seems to be an issue, but I would have to ask here in a different post to confirm so I could try and fix it later.

The example site I have with a quick CSS patch of the issue is a [GitHub pages site](https://fireisgood.github.io/fancy-text/) if you want to look at the proposed transition more.

Since I was simply linking from a CDN, The patch is putting `unset` on those specific elements:

```css
a,
input,
select,
textarea {
  transition: unset;
}
```

**Comment by Leftium** [comment]
Adding a transition to the `<html>` element seems to work, too: https://github.com/Leftium/zbang/commit/6943ad69d5112df1227cff49502fd25210fc6540

I think it looked weird because the `<html>` element's background changed immediately, while other Pico CSS elements took more time to transition.
